server/views/formPages/approvedView.html

```diff
@@ -156,7 +156,7 @@ <h2 class="govuk-heading-m">Other information</h2>
   </div>
 
 {{ govukButton({
-  href: backLink,
+  href: "/",
```

Comment:
`backLink` is not passed in to the template. I can't understand how this was ever a passing test.